salt/modules/state.py

```diff
@@ -285,7 +293,6 @@ def request(mods=None,
         salt '*' state.request test
         salt '*' state.request test,pkgs
     '''
-    kwargs['test'] = True
```

Comment:
@jfindlay Why was this line removed? This change means that a state will run when it is requested, which I don't believe was the intention here.